composer.json

```diff
@@ -20,10 +20,7 @@
         "jakub-onderka/php-console-highlighter": "0.3.*"
     },
     "require-dev": {
-        "phpunit/phpunit": "~3.7|~4.0",
-        "symfony/finder": "~2.1",
-        "squizlabs/php_codesniffer": "~2.0",
-        "fabpot/php-cs-fixer": "~1.3"
```

Comment:
But that workflow requires people to commit things that are wrong. It doesn't empower them to actually do it right the first time. They essentially have to break a build to see that they've done something wrong, and that's not a good experience at all. Travis isn't there as a first defense, it's the backup in case a failing test gets through. StyleCI should be the same.